game/src/Game/GameSession/billboards.tres

```diff
@@ -0,0 +1,11 @@
+[gd_resource type="ShaderMaterial" load_steps=2 format=3 uid="uid://dyeip7yhqf1g7"]
```

Comment:
Is this shader material needed? Would it be possible to use the shader directly without it? We don't have a `tres` file for the map shader, although we may just be creating the material in scripts rather than via the editor like this

Reply:
Removed the material.